History.md

```diff
@@ -1,3 +1,5 @@
+* BMurri: WIXBUG:4532 - Make it an error for InstallPrivileges and InstallScope to specify contradictory values.
```

Comment:
MikeGC had a great idea of creating a file for each history entry instead of manually updating History.md to avoid merge conflicts.  See https://github.com/wixtoolset/wix3/tree/develop/history for details.